src/resources/postcss/tickets-registration-page.pcss

```diff
@@ -36,6 +36,13 @@ body.page-tribe-attendee-registration {
 		}
 	}
 
+	/* -----------------------------------------------------------------------------
+	 * Hide from screenreaders & browsers
+	 * ----------------------------------------------------------------------------- */
+	.tribe-common-a11y-hidden {
```

Comment:
does this not already come from common? or is common CSS not loaded on the tickets registration page?

Reply:
Specifically in the AR page the loader is outside of the wrapper, and it needs this class to be hidden.

Reply:
@paulmskim any further concern here?

Reply:
no concerns. this is good.